packages/create-sitecore-jss/src/templates/nextjs-sxa/src/pages/api/sitemap.ts

```diff
@@ -36,7 +36,7 @@ const sitemapApi = async (
 
     try {
       const fetcher = new NativeDataFetcher();
-      const response = await fetcher.fetch<ReadableStream<Uint8Array>>(sitemapUrl);
+      const response = await fetcher.fetchStream<Uint8Array>(sitemapUrl);
```

Comment:
I think the bug introduced by previous PR was in the [editing-render-middelware](https://github.com/Sitecore/jss/blob/9b0b19aaeb6c756fee9c1a603cb875120d285d3a/packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.ts#L136). You would need to update the reference there too.